packages/initializer-jwt/src/index.ts

```diff
@@ -16,6 +18,18 @@ export interface JWTProps {
 const key = 'jwt'
 export const getProps = () => context.initializerProps<JWTProps>(key)
 
+export const resolveAuthCookieName = (
+  cookieNameConfig?: string | boolean
+): string | null => {
+  if (typeof cookieNameConfig == 'boolean' && cookieNameConfig) {
```

Comment:
the conditional is a bit complicated. try something like this:

`if(cookieNameConfig === true)`